src/main/resources/ValidationMessages.properties

```
@@ -0,0 +1 @@
+value.negative=Email should have @
```

```suggestion
email.value.notcontains.at=Email should have @
```
значення value.negative дуже загальне

---

value.negative.whenUserLogin я змінив на це